src/api/java/de/teamlapen/vampirism/api/event/VampirismVillageEvent.java

```diff
@@ -45,12 +54,14 @@ public IVampirismVillage getVillage() {
         EntityVillager seedVillager;
         private @Nullable
         EntityVillager newVillager;
-        private boolean willBeVampire;
+        private boolean willBeConverted;
+        private final IPlayableFaction<?> faction;
 
-        public SpawnNewVillager(@Nonnull IVampirismVillage village, @Nonnull EntityVillager seedVillager, boolean willBeVampire) {
+        public SpawnNewVillager(@Nonnull IVampirismVillage village, @Nonnull EntityVillager seedVillager, boolean willBeConverted, @Nonnull IPlayableFaction<?> faction) {
```

Comment:
Does the faction have to be non null?
Maybe we want to spawn a villager in a neutral village.
Also, if it is annotated Nonnull, the getter (and the field) should probably be annotated as well

Reply:
until now there is no "village live" without a controlling faction, but now the event will accept it